bibliopixel/remote/control.py

```diff
@@ -114,7 +114,8 @@ def __init__(self, layout, animations,
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
             'get_config': self.get_config,
-            'trigger_animation': self.run_animation
```

Comment:
This line is in the delta because you won't leave trailing commas on your lists and dicts - which is best practice for this reason!